libs/dh/market-participant/actors/feature-actors/src/lib/create/steps/dh-new-actor-step.component.ts

```diff
@@ -27,36 +27,36 @@ import {
 import { VaterStackComponent } from '@energinet-datahub/watt/vater';
 import { WattTextFieldComponent } from '@energinet-datahub/watt/text-field';
 import { WattPhoneFieldComponent } from '@energinet-datahub/watt/phone-field';
-import { WattDropdownComponent, WattDropdownOptions } from '@energinet-datahub/watt/dropdown';
+import { WattDropdownComponent } from '@energinet-datahub/watt/dropdown';
 import { WattFieldErrorComponent, WattFieldHintComponent } from '@energinet-datahub/watt/field';
 import { getGridAreaOptions } from '@energinet-datahub/dh/shared/data-access-graphql';
 
-import { ActorForm } from '../dh-actor-form.model';
+import { DhActorForm } from '../dh-actor-form.model';
 import { dhMarketParticipantNameMaxLength } from '../../dh-market-participant-name-max-length.validator';
 
 @Component({
   standalone: true,
   selector: 'dh-new-actor-step',
   imports: [
-    VaterStackComponent,
+    ReactiveFormsModule,
     TranslocoDirective,
+
+    VaterStackComponent,
     WattTextFieldComponent,
     WattFieldErrorComponent,
     WattFieldHintComponent,
     WattDropdownComponent,
     WattPhoneFieldComponent,
-    ReactiveFormsModule,
     DhDropdownTranslatorDirective,
-    RxPush,
   ],
   styles: [
     `
       :host {
         display: block;
       }
 
-      watt-dropdown {
-        width: 100%;
+      .grid > .grid__item {
```

Comment:
Suggestion: Seems odd to me that we have a flexbox layout, but apply grid to it? Maybe vater should support grid layout as well?

Reply:
We are actually using the vater layout here, but apparently it's not enough. 
As per the screenshot below, the layout is divided into two columns. Selecting grid areas in the dropdown expands the first column because the dropdown doesn't have a specific width. The change above tries to address that. Even though the class is called `grid` there's no `grid` applied to the layout. It's just the name of the class. I can change it to something else if it causes confusion. 

![image](https://github.com/user-attachments/assets/70e4993e-d4d2-443a-83c6-b9b7f2292fc1)

Reply:
another name might be a good idea, to avoid confusion :-)